### DISCUSSION TITLE
Compression at route level

### DISCUSSION THREAD

**Discussion by imcspandey** [kartikk221/hyper-express#234] [General]
is there any way to enable compression on a specific route? 
can we use compression npm package with hyper express? I tried it but did not work.

**Comment by kartikk221** [comment]
Hi, are you talking about compression with WebSockets or HTTP?

**Reply by imcspandey** [reply]
with HTTP. I have seen it's possible with websockets.

**Reply by kartikk221** [reply]
hyper-express / uWS doesn't implement native compression for HTTP. You can utilize a community package to implement your own compression through a middleware If desired by consuming the `Request` as a `ReadableStream` which can be piped to your compression / decompression of choice.
